packages/p2p-v2/types.ts

```diff
@@ -13,9 +13,18 @@ declare global {
 }
 
 export type TAdvertiserPaymentMethods = ReturnType<typeof p2p.advertiserPaymentMethods.useGet>['data'];
-
+export type TPaymentMethods = ReturnType<typeof p2p.paymentMethods.useGet>['data'];
+export type TAccumulatedPaymentMethods = Record<
+    string,
+    NonNullable<TAdvertiserPaymentMethods | TPaymentMethods>[number]
+>;
+export type TPaymentMethod = NonNullable<TAdvertiserPaymentMethods>[number] | NonNullable<TPaymentMethods>[number];
+type TPaymentMethodDisplayName = NonNullable<
+    ReturnType<typeof p2p.paymentMethods.useGet>['data']
+>[number]['display_name'];
```

Comment:
isn't this just a string @nada-deriv ?

```suggestion
type TPaymentMethodDisplayName = NonNullable<
    ReturnType<typeof p2p.paymentMethods.useGet>['data']
>[number]['display_name'];
```

Reply:
yes.. my bad.. i thought it was a set of enums..